ci/install-runtime-deps.sh

```diff
@@ -2,7 +2,11 @@
 
 # This script is meant to be sourced.
 
-case "$CONTAINER_OS" in
+
+OS="$(. /etc/os-release; echo "$ID:$VERSION_ID")"
```

Comment:
So the architecture/flavor is an env.; and OS distro:version is from this local file. 
Why not use the same approach for the arch/flavor as well?

Reply:
`ARCH` is used for cross-compiling. It's not limited to the arch of the build machine.